po/it.po

```diff
@@ -1,6 +1,6 @@
 # Italian translation for GRAMPS
 # This file is distributed under the same license as the Gramps package.
-# Copyright (C) 2001-2013 the Gramps Project
+# Copyright (C) 2001-2019 the Gramps Project
```

Comment:
Please submit the change against the maintenance/gramps50 branch, not master.

But first I'm going to restore the existing translation of the gramplet, which already existed as a single message and you can still see in the comments in the po file, but it was split out in several messages.
After that, this review may not be needed.

Reply:
Hi tosky,
thanks for your review and for pointing out the maintenance/gramps50 branch.